pepper-apis/studybuilder-cli/studies/lms/patches/lms-release-pdfs.conf

```diff
@@ -17,6 +17,11 @@
               "type": "ACTIVITY",
               "activityCode": "MEDICAL_RELEASE",
               "versionTag": "v1"
+            },
+            {
```

Comment:
workaround to restrict uploading both self and pediatric release pdf configs to elastic for all users.